source/guides/stacks/dlrs.rst

```diff
@@ -1007,8 +1009,8 @@ To stop the container, execute the following from your host system:
    .. code-block:: console
 
       # docker rmi 827
-      Untagged: clearlinux/stacks-dlrs-oss:latest
-      Untagged: clearlinux/stacks-dlrs-oss@sha256:381f4b604537b2cb7fb5b583a8a847a50c4ed776f8e677e2354932eb82f18898
+      Untagged: sysstacks/dlrs-tensorflow-clearlinux:latest
+      Untagged: sysstacks/dlrs-tensorflow-clearlinux@sha256:381f4b604537b2cb7fb5b583a8a847a50c4ed776f8e677e2354932eb82f18898
       Deleted: sha256:82757ec1648a906c504e50e43df74ad5fc333deee043dbfe6559c86908fac15e
```

Comment:
Ln 1027L
It's not clear in this header if you mean "OpenMPI", referenced above, or if you mean something like:
`Cluster OpenMP* product for Intel® C++ Compiler for Linux* OS`

Please correct this header.

Reply:
Corrected header and rewrote initial paragraph for clarity. I hope.

Reply:
Got it. Thank you.